content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Installing Ambient Mesh with Helm
+description: How to install Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide shows you how to install Ambient Mesh with Helm.
+Besides the demo in [Getting Started with Ambient Mesh](/docs/ops/ambient/getting-started/),
+we **encourage** you to follow this guide to install Ambient Mesh.
+Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
```

Comment:
I feel like it should be obvious by this point, but we should maybe add a note that we only support Helm 3 and newer.

Reply:
Thanks, I've added the prerequisite step copied from the normal helm installation [https://istio.io/latest/docs/setup/install/helm/.](https://istio.io/latest/docs/setup/install/helm/#prerequisites)

Reply:
+1 on helm 3.x plus

Reply:
Right now we have the requirement of helm version to be 3.6+ in the helm installation guide https://istio.io/latest/docs/setup/install/helm/#prerequisites, shouldn't we use the same value for consistency?